src/store/offers/api-actions.ts

```diff
@@ -46,8 +51,11 @@ export const loadReviews = createAsyncThunk<ReviewListItem[], string, ThunkOptio
 
 export const sendReview = createAsyncThunk<ReviewListItem, ReviewPayload, ThunkOptions>(
   'offer/sendReview',
-  async ({formData, offerId}, { extra: api }) => {
+  async ({formData, offerId, onSuccess}, { extra: api }) => {
     const {data} = await api.post<ReviewListItem>(`${ApiUrl.Reviews}/${offerId}`, formData);
+    if (onSuccess) {
+      onSuccess();
+    }
```

Comment:
можно через optional chaining 
```suggestion
      onSuccess?.();
```